### PR TITLE
Source File: enable high strictness level

### LINKED ISSUE #19076: Source File: enable high test strictness level in SAT
## What
A `test_strictness_level` field was introduced to Source Acceptance Tests (SAT).
File is a generally_available connector, we want it to have a `high` test strictness level.

**This will help**:
- maximize the SAT coverage on this connector.
- document its potential weaknesses in term of test coverage.

## How
1. Migrate the existing `acceptance-test-config.yml` file to the latest configuration format. (See instructions [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/bases/source-acceptance-test/README.md#L61))
2. Enable `high` test strictness level in `acceptance-test-config.yml`. (See instructions [here](https://github.com/airbytehq/airbyte/blob/master/docs/connector-development/testing-connectors/source-acceptance-tests-reference.md#L240))
3. Commit changes on `acceptance-test-config.yml` and open a PR.
4. Run SAT with the `/test` command on the branch.
5. If tests are failing please fix the failing test or use `bypass_reason` fields to explain why a specific test can't be run.

### PR DESCRIPTION
**Pull request by roman-yermilov-gl**
## What
*Enable high strictness level for SAT*

### CODE REVIEW

**Review by bazarnov — APPROVED** [review]

**Review by arsenlosenko — APPROVED** [review]

File: airbyte-integrations/connectors/source-file/acceptance-test-config.yml [hunk 1]
```diff
@@ -17,8 +18,14 @@ acceptance_tests:
...
+          path: "integration_tests/expected_records.jsonl"
+          extra_fields: no
+          exact_order: no
+          extra_records: yes
   full_refresh:
     tests:
       - config_path: "integration_tests/config.json"
         configured_catalog_path: "integration_tests/configured_catalog.json"
+  incremental:
+    bypass_reason: "Incremental syncs are not supported on this connector."
```
Comment (alafanechere) [inline airbyte-integrations/connectors/source-file/acceptance-test-config.yml:31]: Please add missing end lines on your files.

**Review by alafanechere — APPROVED** [review]
Thanks for enabling this! Could you please format your files to add the missing end lines?

File: airbyte-integrations/connectors/source-file/acceptance-test-config.yml (hunk 1 above)
Comment (lazebnyi) [reply airbyte-integrations/connectors/source-file/acceptance-test-config.yml]: updated

**Review by lazebnyi — APPROVED** [review]

**Pull request merged**